cdap-data-fabric/src/test/java/co/cask/cdap/data2/dataset2/lib/FileSetTest.java

```diff
@@ -34,33 +40,56 @@
 
 public class FileSetTest extends AbstractDatasetTest {
 
-  static FileSet fileSet;
-  private static final Id.DatasetInstance testFileSetInstance = Id.DatasetInstance.from(NAMESPACE_ID, "testFileSet");
+  static FileSet fileSet1;
+  static FileSet fileSet2;
+  private static final Id.Namespace NAMESPACE2 = Id.Namespace.from("yourspace");
+  private static final DatasetNamespace dsNamespace = new DefaultDatasetNamespace(CConfiguration.create());
+  private static final Id.DatasetInstance testFileSetInstance1 =
+    dsNamespace.namespace(Id.DatasetInstance.from(NAMESPACE_ID, "testFileSet"));
```

Comment:
In a different namespace, though. I should probably call it ``NAMESPACE_ID2`` or ``OTHER_NAMESPACE_ID`` to make the distinction clearer.